src/utils.test.ts

```diff
@@ -0,0 +1,255 @@
+
+      const expected = "shell";
+      const actual = filterOutVariant(word);
+
+      expect(actual).toBe(expected);
+    });
+
+    it("should return word and tip from word when it has no variant", () => {
+      const word = "fire:f__e";
+
+      const expected = "fire:f__e";
+      const actual = filterOutVariant(word);
+
+      expect(actual).toBe(expected);
+    });
+
+    it("should filter out variant from word when it has no tip", () => {
+      const word = "sheet/paper";
+
+      const expected = "sheet";
+      const actual = filterOutVariant(word);
+
+      expect(actual).toBe(expected);
+    });
+
+    it("should filter out variant from word when it has a tip with character reveal", () => {
+      const word = "ocean/sea:o___n";
+
+      const expected = "ocean:o___n";
+      const actual = filterOutVariant(word);
+
+      expect(actual).toBe(expected);
+    });
+
+    it("should filter out variant from word when it has a tip with several words", () => {
+      const word = "ocean/sea:a lot of water";
+
+      const expected = "ocean:a lot of water";
+      const actual = filterOutVariant(word);
+
+      expect(actual).toBe(expected);
+    });
+  });
+
+  describe(parseWords.name, () => {
+    it("should return empty string when words are undefined", () => {
+      const words = undefined;
+
+      const expected = "";
+      const actualFillIn = parseWords(words, "fillIn");
+      const actualFillInTarget = parseWords(words, "fillIn", "target");
+      const actualDragText = parseWords(words, "dragText");
+      const actualDragTextTarget = parseWords(words, "dragText", "target");
+
+      expect(actualFillIn).toBe(expected);
+      expect(actualFillInTarget).toBe(expected);
+      expect(actualDragText).toBe(expected);
+      expect(actualDragTextTarget).toBe(expected);
+    });
+
+    it("should return empty string when words are empty", () => {
+      const words = "";
+
+      const expected = "";
+      const actualFillIn = parseWords(words, "fillIn");
+      const actualFillInTarget = parseWords(words, "fillIn", "target");
+      const actualDragText = parseWords(words, "dragText");
+      const actualDragTextTarget = parseWords(words, "dragText", "target");
+
+      expect(actualFillIn).toBe(expected);
+      expect(actualFillInTarget).toBe(expected);
+      expect(actualDragText).toBe(expected);
+      expect(actualDragTextTarget).toBe(expected);
+    });
+
+    it("should parse one word correct (fillIn)", () => {
+      const words = "ocean,sjø";
+
+      const expected = "<p>ocean *sjø*</p>";
+      const actual = parseWords(words, "fillIn");
+
+      expect(actual).toBe(expected);
+    });
+
+    it("should parse one word correct (dragText)", () => {
+      const words = "fire,ild";
+
+      const expected = "fire *ild*\n";
+      const actual = parseWords(words, "dragText");
+
+      expect(actual).toBe(expected);
+    });
+
+    it("should parse one word correct with variant (fillIn)", () => {
```

Comment:
Since there are so many tests that are tagged with fillIn or dragText, we _could_ use `describe`, but this works ✨